src/DSP.jl

```diff
@@ -24,6 +21,14 @@ end
 
 @importffts
 
+if VERSION >= v"0.7.0-DEV.602"
+    if isdefined(Base, :DSP)
+        import Base: conv, conv2, deconv, filt, filt!, xcorr
+    else
+        export conv, conv2, deconv, filt, filt!, xcorr
+    end
+end
+
 include("dspbase.jl")
```

Comment:
Totally understand. I'm guessing/hoping that there will be folks from my poster session that might want to try out the audio stuff now that our conference is wrapping up.